openshift_tools/monitoring/zagg_sender.py

```diff
@@ -27,6 +27,14 @@
 import os
 import yaml
 
+# Add Hawkular sender backend (Optional)
```

Comment:
@simon3z :+1: 
a. removed the optional module dependency. (ops will need to install hawkular-client on their systems)
b. sending data to hawkular is still off by default.